libbeat/outputs/codecs/json/json.go

```diff
@@ -0,0 +1,54 @@
+package json
```

Comment:
Probably more a quick follow up discussion so we are on the same page about future package naming an potential refactoring (if we will do it) ;-)